src/Options.php

```diff
@@ -9,29 +9,67 @@ class Options
      *
      * @var array
      */
-    public $fileTypes = [".php"];
+    private $fileTypes = [".php"];
 
     /**
-     * @param \stdClass|null $options
+     * @param \Traversable|\stdClass|array|null $options
      */
-    public function __construct(\stdClass $options = null)
+    public function __construct($options = null)
     {
         // Do nothing when the $options parameter is not an object
-        if (!is_object($options)) {
+        if (!is_object($options) && !is_array($options) && (!$options instanceof \Traversable)) {
```

Comment:
that won't work, you need to do it like this: `!($options instanceof \Traversable)`